seeder.js

```diff
@@ -119,21 +120,49 @@ var rooms = [
     roomPrice: 10000,
     type: "Entire villa",
     numberOfBeds: 6,
-    photoUrl: "https://a0.muscache.com/im/pictures/926b721c-6f9d-4eb1-873f-5e1b43205805.jpg?im_w=1200",
+    imageUrl: "https://a0.muscache.com/im/pictures/926b721c-6f9d-4eb1-873f-5e1b43205805.jpg?im_w=1200",
     savedList: '',
     roomUrl: "https://www.airbnb.com/rooms/40900055?adults=4&children=0&infants=0&source_impression_id=p3_1596749651_Fk1Fycii%2BWu4m5kf",
     superHost: false
   })
 ];
 
+var lists = [
+  new Room.List({
+    listName: "Summer Trips",
+    numberOfItems: 6
+  }),
+  new Room.List({
+    listName: "Europe Trip",
+    numberOfItems: 11
+  }),
+  new Room.List({
+    listName: "Thanksgivings",
+    numberOfItems: 2
+  })
+];
+
 var counter = 0;
 for (var i =0; i < rooms.length; i++) {
   rooms[i].save((err, result) => {
     counter += 1;
     if (err) {
-      console.log('Error seeding data')
+      console.log('Error seeding room data')
```

Comment:
same thing as above, probably want to send the error all the way to the client, not only in the console.